README.md

```diff
@@ -139,6 +139,9 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Hygraph](https://hygraph.com/write-for-hygraph) - Up to $300 per piece
   > Technical tutorials or blogs with code about Hygraph or GraphQL with Jamstack or tooling of your choice.
 
+- [InfoQ](https://www.infoq.com/write-for-infoq/) - Quarterly revenue share
```

Comment:
@synodinos can you explain more about what this means?

We typically only include resources that pay known fees to writers (hence why we don't include Medium or other publications that only give writers a cut of display ads/subscriptions).

Happy to consider this, I just want to be transparent with our writers.